src/main/java/Duke.java

```diff
@@ -1,10 +1,131 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    /*ATTRIBUTES*/
+
+    private static Task[] tasks = new Task[100]; //array to store task list
+    private static int taskCount; //store total number of tasks
+
+
+    /*METHODS*/
+
+    //adds task to the task list
+    public static void addTask(Task t) {
+        printHorizontalLine();
+        tasks[taskCount] = t;
+        t.printTaskNotif();
+        System.out.println("Now you have " + (taskCount + 1) + " tasks in the list");
+        printHorizontalLine();
+
+        taskCount++;
+    }
+
+    //prints task list when "list" is keyed by user
+    public static void printList() {
+        printHorizontalLine();
+        System.out.println("Here are the tasks in your list:");
+        for (int i = 0; i < taskCount; i++) {
+            String type = tasks[i].type;
+            String icon = tasks[i].getStatusIcon();
+            System.out.println((i + 1) + "." + "[" + type + "]" + " [" + icon + "] " + tasks[i].description);
+        }
+        printHorizontalLine();
+    }
+
+    //prints specific task that is done
+    public static void printDoneTask(Task t) {
+        printHorizontalLine();
+        System.out.println("Nice, I've marked this task as done:");
+        String type = t.type;
+        String icon = t.getStatusIcon();
+        System.out.println("[" + type + "]" + " [" + icon + "] " + t.description);
```

Comment:
Put icon and type in the Deadline, Event and ToDo classes. So dont have to repeat getStatusIcon and type multiple times